### ISSUE TITLE
Unable to replicate the Runtime pcc drop in mixer_b16_224.goog_in21k ONNX model with op level sanity

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-forge-fe#2927]
## Problem Description

- The test `forge/test/models/onnx/vision/mlp_mixer/test_mlp_mixer_onnx.py::test_mlp_mixer_timm_onnx[mixer_b16_224.goog_in21k]` fails with Tensor mismatch. PCC = 0.6577629768931035 at runtime.

### Unique Op Test

- Running unique op tests (generated by scripts/model_analysis/generate_models_ops_test.py) revealed a tensor mismatch: PCC = 0.942156523210061 in Conv2D.
```
Note: Random inputs are passed  
forge/test/models/onnx/vision/mlp_mixer_debug/mlp_ops/test_conv2d.py::test_module[inference-Conv2D0-[((1, 3, 224, 224), torch.float32)]]
```
- The model contains only one Conv2D [here](https://github.com/huggingface/pytorch-image-models/blob/072783331f42dd320e3c6fe890a5c197fc21958f/timm/layers/patch_embed.py#L59C21-L59C31).
- Cross-verification with actual inputs (using a sliced model returning the output after Conv2D) passed successfully.
- So the issue is related to random inputs, not the actual model input.

### Model Without Weights

- The model passes with valid PCC when pretrained=False.
- This indicates that the issue lies in weights, not the architecture.
- However, pretrained weights are necessary for meaningful results, so debugging proceeds further.

### Model Slicing

- Flow: `Stem → [MixerBlock ×12] → Norm → Head`
- A significant PCC drop was observed after Norm: Drop from ≥0.99 → 0.31555294422564506 after this [normalisation](https://github.com/huggingface/pytorch-image-models/blob/072783331f42dd320e3c6fe890a5c197fc21958f/timm/models/mlp_mixer.py#L341)
- Attempted Norm sanity test with saved inputs (captured during forward pass), but unable to replicate the drop
- **Conclusion:** The drop occurs only in the combination of Norm + specific preceding block ops.

#### Experiments: PCC Drop Across Blocks

| Block Range   | PCC                  |
|---------------|----------------------|
| Stem → Norm   | 0.3275189391333936   |
| b0 → Norm     | 0.327518939133393    |
| b5 → Norm     | 0.3349051399160558   |
| b8 → Norm     | 0.4017389784957895   |
| b10 → Norm    | 0.3658431045481776   |
| b11 → Norm    | 0.9891757004018408   |

- The drop almost disappears when block 10 is skipped.
- The minimal failing case is b10 + b11 + Norm.
- Further slicing of b10 could reduce the small number of ops, but since b11 sits between b10 and Norm, it blocks direct reduction to op-level sanity.

### Key Observations

- All 12 MixerBlocks are architecturally identical.
- The only difference lies in their weights.
- As noted earlier, disabling weights (pretrained=False) allows the model to pass.
- This makes debugging trickier, since the issue is not with the ops, but with specific weight values interacting with them.

### Repro 

```
git checkout kkannan/sep3_mlp_onnx_pcc_issue
git submodule update --init --recursive
git lfs pull --include 'b0_ip.pt b10_ip.pt b11_ip.pt b1_ip.pt b2_ip.pt b3_ip.pt b4_ip.pt b5_ip.pt b6_ip.pt b7_ip.pt b8_ip.pt b9_ip.pt norm_ip.pt s_ip.pt'

# Whole model run 
pytest forge/test/models/onnx/vision/mlp_mixer_debug/test_mlp_mixer_onnx.py -svv

# norm sanity
pytest forge/test/models/onnx/vision/mlp_mixer_debug/test_s1.py

# Block sanity(b10 + b11 + Norm)
pytest forge/test/models/onnx/vision/mlp_mixer_debug/test_s6.py

```
